rotkehlchen/tests/api/test_users.py

```diff
@@ -179,9 +194,12 @@ def test_user_creation_with_no_analytics(rotkehlchen_api_server, data_dir):
     ),
 )
 def test_user_creation_permission_error(
-        mock_path_mkdir, rotkehlchen_api_server,  # pylint:disable=unused-argument
-):
-    """Test that creating a user when data directory permissions are wrong is handled"""
+        mock_path_mkdir: mock.MagicMock,
+        rotkehlchen_api_server: 'APIServer',
+        use_clean_caching_directory: bool,
+        start_with_logged_in_user: bool,
+) -> None:
+    """Test that creati: Path,ng a user when data directory permissions are wrong is handled"""
```

Comment:
Why did you do here?
```suggestion
    """Test that creating a user when data directory permissions are wrong is handled"""
```